app/views/merchant/dashboard/index.html.erb

```diff
@@ -1 +1,6 @@
 <%= content_tag :h1, "Welcome #{current_user.name}!!" %>
+
+<%= @merchant.name %>
+<%= @merchant.address %>
+<%= "#{@merchant.city}, #{@merchant.state}" %>
```

Comment:
I like the way you interpolated this to be `city, state` rather than 2 separate lines for each